docker/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+set -e
+
+# Remove a potentially pre-existing server.pid for Rails.
+rm -f /usr/src/app/tmp/pids/server.pid
+
+echo "bundle install..."
+bundle check || bundle install --jobs 4
+
```

Comment:
for easy docker composing, kicking this in the entrypoint is handy
```suggestion
bundle exec rake db:create db:migrate
```